### DISCUSSION TITLE
Welcome to Vertical_Velocities_SOP Discussions

### DISCUSSION THREAD

**Discussion by fmargirier** [OceanGlidersCommunity/Vertical_Velocities_SOP#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  * Feel free to discuss the name of the vertical velocity variables
  * Please share your necessary navigation parameters to be shared for the compuatation of glider flight models

**Comment by eleanorfrajka** [comment]
Hi folks - now I'm not sure if we're using the GitHub discussions or the UG2 slack to discuss things.  But we'll aim to have some data in OG format for the second half of the year.  One thing I didn't yet check was whether or not there's a format checker for OG.

**Reply by izzatnadzmi** [reply]
Hello! Just checking in to say that this is a super cool effort and will check out the space for updates (through this unofficial Git account of mine).
Currently at NOC working on a AUV/glider-ocean model simulator. I'm particularly interested in the glider dynamic/flight models being used in the community (especially the ones used for infering vertical velocities), including the interfaces, parameters, and formats for the models and datasets. This would be helpful for us to incorporate models of interest for the scientists. (I've also checked out the DACs SOP!)
